services/app-web/src/hooks/useOTEL.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
+import { Span } from '@opentelemetry/api'
+
+import { useAuth } from '../contexts/AuthContext'
+import { useCurrentRoute } from './useCurrentRoute'
+import { getTracer } from '../otelHelpers'
+
+// Adds a new OTEL span for each route change with some custom attributes. Proxy to users time on a given page.
+export const useOTEL = (title?: string): void => {
+    const { currentRoute } = useCurrentRoute()
+    const { loggedInUser } = useAuth()
+
+    useEffect(() => {
+        const tracer = getTracer()
+
+        const span: Span = tracer.startSpan('React: Page View')
+        span.setAttribute('current_route', currentRoute.toString())
```

Comment:
I think we might want to use the `SemanticAttributes` to record the keys on these attributes. It's a spec of constants that OTEL is standardizing upon to make it easier to surface common keys. [Here's a list](https://open-telemetry.github.io/opentelemetry-js/modules/_opentelemetry_semantic_conventions.html) from the JS lib.

So for `user_id` we'd use `[SemanticAttributes.ENDUSER_ID]: loggedInUser.email`. They don't exist for everything, but where there is a standardized key I think we should adopt it.